### PR TITLE
[MOB-5898] - Encryption Flag with 3.4.11

### PR DESCRIPTION
**Pull request by Ayyanchira**
## 🔹 JIRA Ticket(s) if any

* [MOB-5898](https://iterable.atlassian.net/browse/MOB-5898)

## ✏️ Description

> Pointing to latest Android version with encryption flag enabled

[MOB-5898]: https://iterable.atlassian.net/browse/MOB-5898?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: ts/__tests__/Iterable.spec.ts [hunk 1]
```diff
@@ -239,6 +239,7 @@ test('iterableConfig_noParams_defaultValues', () => {
   expect(config.allowedProtocols).toEqual([])
   expect(config.androidSdkUseInMemoryStorageForInApps).toBe(false)
   expect(config.useInMemoryStorageForInApps).toBe(false)
+  expect(config.encryptionEnforced).toBe(false)
 })
 
 test('iterableConfig_noParams_defaultDictValues', () => {
```
Comment (evantk91) [inline ts/__tests__/Iterable.spec.ts:245]: Missing the encryptionEnforced in the next test.

**Review by evantk91 — COMMENTED** [review]
Missing encryptionEnforced in the default dict values test.

File: ts/__tests__/Iterable.spec.ts (hunk 1 above)
Comment (Ayyanchira) [reply ts/__tests__/Iterable.spec.ts:245]: Nice catch 👍🏼

**Review by roninopf — APPROVED** [review]

**Review by evantk91 — APPROVED** [review]
LGTM

**Pull request merged**